tests/PyPI-pytest/run_all.py

```diff
@@ -0,0 +1,178 @@
+            )
+            
+            stdout, stderr = process.communicate()
+            exit_pytest = process.returncode
+            print("stdout: \n", stdout)
+            print("stderr: \n", stderr)
+            
+    except Exception as exceptObj:
+        print(exceptObj)
+        continue
+
+
+    os.chdir(base_dir)
+
+    try:
+        # get wheel pytest results
+        with withVirtualenv("wheel_venv_%s" % package_name) as venv:
+            venv.runCommand(
+                commands=[
+                    "call Scripts/activate",
+                    "python -m pip install pytest",
+                    "git clone https://github.com/%s/%s.git" % (package_name,package_name),
+                    "git clone https://github.com/nuitka/nuitka.git",
+                    "cd nuitka",
+                    "python setup.py develop",
+                    "cd ../%s" % package_name,
+                    "python -m pip install -r dev-requirements.txt",
+                    "python setup.py bdist_nuitka",
+                    "python -m pip uninstall -y %s" % package_name,
+                ]
+            )
+
+            dist_dir = os.path.join(base_dir, "wheel_venv_%s" % package_name, package_name, "dist")
+
+            venv.runCommand(
+                'pip install "%s"'
+                % (os.path.join(dist_dir, os.listdir(dist_dir)[0]))
+            )
+
+            os.chdir(os.path.join(base_dir, "wheel_venv_%s" % package_name, package_name))
+
+            process = subprocess.Popen(
+                args=[
+                    os.path.join(
+                        venv.getVirtualenvDir(),
+                        "bin" if os.name != "nt" else "scripts",
+                        "python",
+                    ),
+                    "-m",
+                    "pytest",
+                    "--disable-warnings",
+                ],
+                stdout=subprocess.PIPE,
+                stderr=subprocess.PIPE, 
+            )
+
+            w_stdout, w_stderr = process.communicate()
+            w_exit_pytest = process.returncode
+            print("stdout: \n", stdout)
+            print("stderr: \n", stderr)
+    
+    except Exception as exceptObj:
+        print(exceptObj)
+        continue
+
+
+    os.chdir(base_dir)
+    # remove venv directory
+    # currently not removing the directories?
+    # removeDirectory(os.path.join(base_dir, "venv_%s" % package_name), ignore_errors=False)
+    # removeDirectory(os.path.join(base_dir, "wheel_venv_%s" % package_name), ignore_errors=False)
+
+
+    # compareOutput
+    '''
+    exit_code_stdout = compareOutput(
+        "stdout",
+        stdout,
+        w_stdout,
+        ignore_warnings=True,
+        ignore_infos=True,
+        syntax_errors=True,
+    )
+
+    exit_code_stderr = compareOutput(
+        "stderr",
+        stderr,
+        w_stderr,
+        ignore_warnings=True,
+        ignore_infos=True,
+        syntax_errors=True,
+    )
+    '''
```

Comment:
I think I need some pointers on how to use `compareOutput` before I can move on. It compares cpython output to nuitka output but in my case I'm trying to compare the regular pytest with the nuitka wheel pytest?

Reply:
Yes, I think you are, but py.test output also contains stack traces, and these will often contain Python code, or object names, that you will benefit from being made comparable.

I suggest we retry having a conference call. I will ping you about it.